client/my-sites/plugins/plugins-navigation-header/index.jsx

```diff
@@ -115,34 +115,39 @@ const PluginsNavigationHeader = ( { navigationHeaderRef, categoryName, category,
 	const { localizePath } = useLocalizedPlugins();
 
 	useEffect( () => {
-		const items = [
-			{
-				label: translate( 'Plugins' ),
-				href: localizePath( `/plugins/${ selectedSite?.slug || '' }` ),
-				id: 'plugins',
-				helpBubble: translate(
-					'Add new functionality and integrations to your site with plugins.'
-				),
-			},
-		];
+		if ( breadcrumbs?.length === 0 || ( ! category && ! search ) ) {
+			dispatch( resetBreadcrumbs() );
```

Comment:
If you go to Activity Log -> Plugin, then use the side menu to click back to Plugins, breadcrumbs?.length > 0, so we need to check that category and search are empty. If they are we can `resetBreadcrumbs()` to start fresh and set 'Plugins' as the root breadcrumb.

Reply:
Should we `resetBreadcrumbs()` as part of visiting any root page? Maybe we can hook this into the controller of those pages?

Reply:
I think that sounds like a good idea. How would feel if we shipped this PR and created a follow up issue for that?

Reply:
Follow up [issue here](https://github.com/Automattic/wp-calypso/issues/68114). I'll have a look at this and e2e tests this 20% cycle. ☺️